### PR TITLE
new python versions support update

### PR DESCRIPTION
**Pull request by TheAmid**
CHANGE:

Changed async=True to background=True

CHECK:

Project works with python version up to 3.8.6 and MATLAB 2020b. Tested on Windows 10 1809

### CODE REVIEW

**Comment by espdev** [comment]
@TheAmid What is minimal version of Matlab is supported after these changes?

Also we can drop Python 3.5 support because 3.5 is no longer supported officially.
https://pythoninsider.blogspot.com/2020/10/python-35-is-no-longer-supported.html

We need to specify minimal Python and Matlab versions which we support.

**Comment by TheAmid** [comment]
The minimal version of Matlab is R2017b with Python 3.6

**Comment by espdev** [comment]
> 
> 
> The minimal version of Matlab is R2017b with Python 3.6

Have you tested "Matlab is R2017b with Python 3.6" with `background=True` instead of `async=True`?

**Comment by TheAmid** [comment]
Good question. I have tested Matlab R2017b + Python 3.6 and everything works fine.

**Review by espdev — APPROVED** [review]

**Comment by espdev** [comment]
@TheAmid Could you change minimal Python version here:
https://github.com/OpenNFT/OpenNFT/blob/master/setup.py#L25

Also you can specify minimal MATLAB version in `setup.py` and check it when installing Matlab engine for Python.

**Comment by TheAmid** [comment]
I edited supported Python versions and added Vatlab version requirements. 

But I'm not sure it is necessary for Matlab. Its version depends on Python and <R2017b versions won't work with "newest" Python 3.6.

File: setup.py [hunk 1]
```diff
@@ -181,6 +182,7 @@ def run(self):
     version=get_version(),
 
     python_requires=PYTHON_REQUIRES,
+    matlab_requires=MATLAB_REQUIRES,
```
Comment (espdev) [inline setup.py:185]: It doesn't work that way. You need to implement Matlab check version in code where we find and install matlab engine Python package.

Somewhere in this function:

https://github.com/OpenNFT/OpenNFT/blob/e8caef4a8020b46aaeb28d06166aa7e634e3bc69/setup.py#L83

**Review by espdev — CHANGES_REQUESTED** [review]
Implement correct check of Matlab version.

**Review by espdev — APPROVED** [review]

**Comment by ykoush** [comment]
closes #76

**Pull request merged**